blog/src/components/Featured/CardWide.jsx

```diff
@@ -0,0 +1,54 @@
+import images from "../../utils/images";
+import "../../styles/Card.scss";
+import PropTypes from "prop-types";
+
+
+const CardWide = ({ blogItemData }) => {
+    const MAX_LENGTH = 250;
+
+  return (
+    <div className="card-grid grid">
+      <div className="card-grid-img">
+        <img data-testid='FeaturedImg' src = {`${blogItemData.image}`} alt = "Featured image" className="object-fit-cover" />
+      </div>
+      <div className="card-grid-text">
+        <div>
+          <div className="badge badge-primary">featured</div>
+          <h2 data-testid='headingfeatured' className="title-lg title featured-title">
+            { blogItemData.title }
+          </h2>
+          
+            {blogItemData.content.length > MAX_LENGTH ?
+            (<p className="text-lg text" data-testid='featureSummary'>
+                    {`${blogItemData.content.substring(0, MAX_LENGTH)}...`}<a href="#">Read more</a>
```

Comment:
Add the blog url to the blog object so that later on when integrating backend user can click on this button and navigate to the blog.